x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/monitor_enabled.tsx

```diff
@@ -51,17 +59,24 @@ export const MonitorEnabled = ({ id, monitor, reloadPage, initialLoading }: Prop
       {isLoading || initialLoading ? (
         <EuiLoadingSpinner size="m" />
       ) : (
-        <EuiSwitch
+        <SwitchWithCursor
           compressed={true}
           checked={enabled}
           disabled={isLoading || isDisabled}
           showLabel={false}
           label={enabled ? labels.DISABLE_MONITOR_LABEL : labels.ENABLE_MONITOR_LABEL}
           title={enabled ? labels.DISABLE_MONITOR_LABEL : labels.ENABLE_MONITOR_LABEL}
           data-test-subj="syntheticsIsMonitorEnabled"
+          isSwitchable={isSwitchable}
           onChange={handleEnabledChange}
         />
       )}
     </>
   );
 };
+
+const SwitchWithCursor = euiStyled(EuiSwitch)<{ isSwitchable: boolean }>`
+  & > button {
+    cursor: ${({ isSwitchable }) => (isSwitchable ? undefined : 'not-allowed')};
+  }
+`;
```

Comment:
Because marking **EuiSwitch** as _disabled_ on the whole table while the data is loading, was making the whole UI flip flop between states. Making only the cursor to `no-allowed` is sort of a workaround to that. We could have a better UX for this once we allow bulk actions on the table.

https://user-images.githubusercontent.com/2748376/182675175-50096535-173d-4b2d-be3d-ebb58be64324.mov